src/front/glsl/types.rs

```diff
@@ -225,6 +225,18 @@ pub const fn type_power(kind: ScalarKind, width: Bytes) -> Option<u32> {
 }
 
 impl Parser {
+    /// Resolves the types of the expressions until `handle` (inclusive)
+    ///
+    /// This needs to be done before the [`typifier`][typifier] can be queried for
```

Comment:
You can just say 

    [`typifier`]

and then at the bottom say:

    [`typifier`]: Context::typifier

There are similar cases elsewhere in this file.

Reply:
Thanks didn't know about this